pom.xml

```diff
@@ -53,7 +53,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
+        </plugins>
     </build>
     <dependencies>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarsource.org/coding_rules#rule_key=xml%3AIndentCheck)